pytentiostat/reporter.py

```diff
@@ -1,7 +1,11 @@
 import pandas as pd
+import shutil
 
+from config_reader import get_output_params
 
-def save_data_to_file(data, filename="Place_Holder.csv"):
+filename, export_file_destination = get_output_params()
```

Comment:
we should maybe think a bit about what behavior we want.  Right now, I think files will be overwritten without the user editing the config file, which may not be the desired behavior.

One way around is to build a file-name on the fly from other metadata...for example have the user put a filename in the config file, but append a time-stamp before writing.  If we capture info such as sample-name the filename could be ``sample-name_timestamp`` or sthg.

Reply:
maybe hvae ``get_output_params`` build the filename and path and hand it back built?

Reply:
- Output now adds timestamp to file name now (H_M_S)